src/lms-ds-loader/lms_ds_loader/argparser.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import configargparse
+from lms_ds_loader.constants import Constants
+from lms_ds_loader.arguments import Arguments
+
+
+def parse_arguments(args_in) -> Arguments:

```

Comment:
What do you guys think - better to have this bare function here, or move this to the Arguments class as a static method?

Reply:
I like the current approach don't see a lot of value on adding this to the Arguments class.

Reply:
This seems like a Pythonic approach. Just feels a little weird having this function without a class 😄 . In C# I definitely would not have a separate class for this function - it would be a builder method on the `Arguments` class. But in Python we have this flexibility. I'm not sure which model is better for future maintenance, really they feel about equal to me. I do not yet have a strong feeling, even though it feels a bit strange leaving this here.